projects/storefrontlib/cms-components/product/product-images/product-images.component.html

```diff
@@ -1,5 +1,13 @@
 <ng-container *ngIf="mainImage$ | async as main">
-  <cx-media [container]="main"></cx-media>
+  <cx-media
+    *cxFeature="'useExtendedMediaComponentConfiguration'"
+    [elementType]="'img'"
+    [container]="main"
+  ></cx-media>
+  <cx-media
+    *cxFeature="'!useExtendedMediaComponentConfiguration'"
+    [container]="main"
+  ></cx-media>
```

Comment:
`query:` this usage of cx-media doesn't contain format property. Is the feature toggle wrapper required?